tc/core/polyhedral/scop.h

```diff
@@ -437,6 +438,13 @@ struct Scop {
       isl::schedule_constraints constraints,
       const SchedulerOptionsView& schedulerOptions);
 
+ public:
+  // Do the simplest possible dependence analysis.
+  // Compute all RAW, WAR, and WAW dependences, and save them in dependences.
+  void computeAllDependences();
+  // Save the RAW, WAR, and WAW dependences
+  isl::union_map dependences;
```

Comment:
The styling policy is to put members below all functions.  In this case, let's put this below read/write map in line 484.